src/qw/cli.py

```diff
@@ -1,30 +1,118 @@
+"""
+The qw (Quality Workflow) tool helps enforce regulatory compliance
+for projects managed on github.
+"""
+
+import json
+import os
+import sys
 from typing import Annotated
 
-import typer as typer
+import git
+import typer
 from loguru import logger
 
-# Main entry point into the application
+import qw.service
+from qw.base import QwError
+
 app = typer.Typer()
 
 
 @app.command()
-def parrot(
-        example: Annotated[str, typer.Argument(help="String to print to user")]
+def init(
+    repo: Annotated[
+        str | None,
```

Comment:
typer wants the Optional syntax rather than the newer union
```suggestion
        Optional[str],
```

Reply:
Yeah, and ruff puts it back again. Optional is correct so I've disabled the check in cli.py

Reply:
ah its because I've set up the target version of python 3.11, Probably a bit too keen for only super modern python. could change `target-version = "py39"` in pyproject.toml for ruff config?

Reply:
I think it's Typer that can't cope with it, so that won't help.

Reply:
nah it will, because typer is relying in the pre `3.10` annotation of `Optional`, if we set the ruff target version then that'll line up with what version typer is targetting as well. I guess it depends on how restructive we want to be with users being able to run `qw`